src/check_jsonschema/catalog.py

```diff
@@ -4,7 +4,7 @@
 
 
 def _bitbucket_pipelines_url() -> str:
-    return "https://bitbucket.org/atlassianlabs/intellij-bitbucket-references-plugin/raw/master/src/main/resources/schemas/bitbucket-pipelines.schema.json"  # noqa: E501
```

Comment:
This schema hosted in the repo of Bitbucket Linky plugin for IntelliJ is deprecated and will eventually be removed (I'm the maintainer of that plugin).